fvm/handler/programs_test.go

```diff
@@ -39,15 +39,15 @@ func Test_Programs(t *testing.T) {
 
 	contractA0Code := `
 		pub contract A {
-			pub fun hello(): String {
+			pub fun hello(): Name {
```

Comment:
?

Reply:
[Fixed]Goland change String and Type in a bunch of unrelated places when I renamed the fields on the MsgAuthConfig